spec/unit/jobs/pollable_job_wrapper_spec.rb

```diff
@@ -59,6 +59,20 @@ class BigException < StandardError
           expect(job_record.resource_type).to eq('droplet')
           expect(job_record.cf_api_error).to be_nil
         end
+
+        context 'when the job defines its state' do
+          before do
+            job.define_singleton_method(:pollable_job_state) do
+              'ABRACADABRA'
+            end
```

Comment:
I changed this from `job.class.define_method` to `job.define_singleton_method` as jobs of this class are actually used in other tests in the file and defining the method can break those tests if they run in a different order.  `define_singleton_method` will define the method only for that specific instance (ruby is nuts).